src/features/academic/components/progress/RequiredCompletionSection.module.scss

```
@@ -0,0 +1,56 @@
+@use '@/styles/color.scss' as color;
+@use '@/styles/spacing.scss' as spacing;
+@use '@/styles/typography.scss' as typography;
+@use '@/styles/radius.scss' as radius;
+
+.title {
+  @include typography.body-md;
+  @include typography.font-weight-bold;
+  margin-top: spacing.$space-28;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)